config/conference.ts

```diff
@@ -87,7 +87,9 @@ const Conference: IConference = {
   GoogleAnalyticsId: 'UA-60040308-1',
   TicketPrice: '$60',
   ChildcarePrice: '$30',
-  EventbriteId: null,
+  TicketsProviderId: TicketsProvider.Tito,
+  TicketsProviderAccountId: 'dddperth',
+  EventId: 'testevent',
```

Comment:
This needs to be `2019`, happy to merge with this for now and we can PR it later